src/external/bitcoin.rs

```diff
@@ -35,12 +36,12 @@ impl AddressInfo {
         };
         let address: AddressInfo = reqwest::get(&request_url)
             .await
-            .expect("Failed to send request")
+            .context("Failed to send request")?
```

Comment:
I meant this kinds of context. If the message contains some info about the function context it is much easier to track down. Otherwise you need to scan the whole stack trace.